src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Ressources/CertificateSchema.json

```diff
@@ -28,30 +28,31 @@
       "title": "Date of birth",
       "description": "Date of Birth of the person addressed in the DGC. ISO 8601 date format restricted to range 1900-2099",
       "type": "string",
-      "pattern": "(19|20)\\d{2}-\\d{2}-\\d{2}",
+      "__format": "date",
+      "pattern": "(19|20)\\d{2}-\\d{2}-\\d{2}(T00:00:00)?",
```

Comment:
I think our Formatter in `HealthCertificate.swift` using `.justLocalDateFormatter` might fail with "T00:00:00", have you tried this?

Reply:
You can try https://github.com/eu-digital-green-certificates/dgc-testdata/blob/main/BG/png/BG-1.png to test it. It should display correctly. If it crashes, then the formatting must be adjusted.

Reply:
Yes, I tested it. It works.